content/en/news/releases/1.17.x/announcing-1.17/_index.md

```diff
@@ -0,0 +1,75 @@
+{{< /tip >}}
+
+## What's new
+
+Since the 1.16 release we’ve added some important new features and marked some of our existing features as Beta signaling that they’re ready for production use. Here are some highlights:
+
+### Canary upgrade and revision tags was promoted to Beta
+
+Basic support for upgrading the service mesh following a canary pattern using revisions was introduced in Istio 1.6 release. Using this approach, you can run multiple control planes side-by-side without impacting an existing deployment and slowly migrate workloads from the old control plane to the new. In Istio 1.10, revision tags was introduced as an improvement to canary upgrades to help reduce the number of changes an operator has to make to use revisions, and safely upgrade an Istio control plane. This is a very widely adopted and used feature by our users in production. All integration tests and end-to-end tests covering documentation have been completed for this feature to graduate to Beta.  
+
+### Helm installation was promoted to Beta
+
+Helm based installation of Istio was first introduced in Istio 0.4, has graduated to Beta. It is one of the most widely used methods to install Istio in production. All requirements to promote this feature to Beta was completed in this release including updating integration tests to use helm charts for install/upgrade, updating Helm integration tests and documenting advanced Helm chart customization and attributes in `values.yaml`.
+
+### Enhancements to Istio dual-stack feature
+
+Added support to generate configuration for Istio gateway with dual stack enabled. `IPv6` support in dual-stack mode was added in Kubernetes in version 1.16 and graduated to stable in the 1.22 release. Istio dual stack support is an alpha feature and was introduced in release 1.16. This is an area under [active development]( https://github.com/istio/istio/issues/40394).
+
+### Added support for filter patching in Istio
+
+Added support for listener filter patching which enables users to perform `ADD`, `REMOVE`, `REPLACE`, `INSERT_FIRST`, `INSERT_BEFORE`, `INSERT_AFTER` operations for `LISTENER_FILTER` in `EnvoyFilter`. 
+
+### Added support for using `QuickAssist Technology` (QAT) `PrivateKeyProvider` in Istio
+
+Added support for using `QuickAssist Technology` (QAT) `PrivateKeyProvider` in SDS and added corresponding configuration for selecting QAT private key provider for gateways and sidecars. This builds on the fact that Envoy added [support for QAT]( https://github.com/envoyproxy/envoy/issues/21531) as another private key provider in addition to [CryptoMB]( https://istio.io/latest/blog/2022/cryptomb-privatekeyprovider/). For more information on QAT, you can refer [here]( https://www.intel.com/content/www/us/en/developer/articles/technical/envoy-tls-acceleration-with-quickassist-technology.html).
+
+### Enhancements to the `RequestAuth` API
+
+Added support to copy JWT claims to HTTP request headers in the `RequestAuth` API.
```

Comment:
Is this new? Feels like this has been around for a little while? @diemtvu do you know?

Reply:
yeah, this is new.https://github.com/istio/istio/issues/39724 (this could be done using a custom EnvoyFilter but with this change, it is included in the RequestAuth API.